exasol/ds/sandbox/lib/dss_docker/Dockerfile

```diff
@@ -2,3 +2,4 @@ FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install --no-install-recommends --assume-yes python3 python3-pexpect
 EXPOSE 8888/tcp
+VOLUME /root/notebooks
```

Comment:
you need to add this in python, because ansible defines this directory

Reply:
This was exactly my major question and concern.

Reply:
See [commit #ba83f7f](https://github.com/exasol/data-science-sandbox/pull/117/commits/ba83f7feff4d3d8883ce636bca2834560d67d562)  for changed implementation.